download_list.json

```diff
@@ -15,19 +15,19 @@
 	  {"file":"genesis_plus_gx_libretro.dll.zip", "url":"http://buildbot.libretro.com/nightly/windows/x86_64/latest/genesis_plus_gx_libretro.dll.zip"},
 	  {"file":"snes9x_libretro.dll.zip", "url":"http://buildbot.libretro.com/nightly/windows/x86_64/latest/snes9x_libretro.dll.zip"},
 	  {"file":"N-Warp Daisakusen V1.1.smc", "url":"https://github.com/OpenEmu/OpenEmu-Update/raw/master/Homebrew/SNES/N-Warp%20Daisakusen/N-Warp%20Daisakusen%20V1.1.smc"},
-	  {"file":"Marilyn_In_the_Magic_World_(010a).7z","url":"http://psxdev.net/homebrew/files/Marilyn_In_the_Magic_World_(010a).7z"},
+	  {"file":"marilyn_v0.12a.7z","url":"https://www.ps2-home.com/media/brew/marilyn_v0.12a.7z"},
```

Comment:
Nice find!

Reply:
I think you can reject the pull and sorry for the trouble
I was looking to help but in a hurry and made some mistakes

Reply:
Hey PRElias - thanks for your help, I've made a bunch of changes and everything now works!